### ISSUE TITLE
cycle marker colours instead of reassigning them

### ISSUE THREAD

**Issue by o-smirnov** [Mulan-94/polarvis#10]
You currently have most recently clicked LoS in purple, previous one in orange, and the least recent one in blue, right? So the "colour" of each plot changes as it moves down the stack. I think it would be more intuitive if each LoS retained the same colour marker while it was visible. So e.g. here:

![image](https://user-images.githubusercontent.com/6470079/110300706-dca32700-7fff-11eb-8cef-d46344a4ee5f.png)

...when the next LoS is clicked, it should be blue... and the next one, orange. 

An even more ambitious idea (this is for @ericmandel to tell me if it's even possible) would be to draw arrows from the markers pointing toward the respective plots. But that would require for the tail of the arrow to be anchored to a world coordinate position, while the head of the arrow is at a physical screen coordinate, so maybe this is crazy...

**Comment by ericmandel** [comment]
I can't think of a good way to mix wcs and display coordinates at either end of an arrow (unless you want to figure  out the angle from the region to the target plot, and recalculate each time the user changes the zoom, pan, rotation ... yuk!) But how about matching the color of the plot's data points to the color of the associated region on the display? That might make for a stronger visual cue ...

**Comment by o-smirnov** [comment]
Yeah, I suspected it was going to be yuk...

> But how about matching the color of the plot's data points to the color of the associated region on the display?

Good idea... wonder it it's possible though, given that the plots are pre-generated offline. @Mulan-94, is there a way to force bokeh to change the colour of the markers on the fly? If so, Eric's idea might work quite well (at least for the frac pol plots, where you only have the one marker... not sure it will fly on the other plots!)

**Comment by o-smirnov** [comment]
> at least for the frac pol plots, where you only have the one marker

Actually wait no, that plot also has up to four marker types which you're already using colour to distinguish, so Eric's suggestion might not work so well after all.

But speaking of distinguishing, the QU plot uses two very similar marker colours, could try some more distinctive ones?

**Comment by ericmandel** [comment]
Last suggestion, given my ignorance of the whole setup: you make the CSS border of each plot match the region color

**Comment by o-smirnov** [comment]
Ah I like that idea, let's see what that looks like! @Mulan-94 currently has a coloured "squash ball" in the title bar of each plot, but indeed a border would make it more obvious.

**Comment by Mulan-94** [comment]
![image](https://user-images.githubusercontent.com/12611059/110475736-ef922600-80e9-11eb-97a3-352613f01ff5.png)

**Comment by o-smirnov** [comment]
Yes, this is very nice. And I can see it's not redrawing the two older plots anymore. Very smooth! Push this version to the public site I say.
